<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Alex Peterson</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spacescape - untitled.xml*</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Spacescape.qrc">
    <normaloff>:/spacescape/images/spacescape-logo.ico</normaloff>:/spacescape/images/spacescape-logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow, QMenuBar, #centralWidget,  #centralWidget QSplitter::handle, QMenuBar QMenu, QMenuBar QMenu::item, QStatusBar {
color: rgb(200, 200, 200);
background-color: rgb(50, 50, 50);
}

QMenuBar {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(104, 104, 104, 255), stop:0.464503 rgba(104, 104, 104, 255), stop:0.46856 rgba(90, 90, 90, 255), stop:1 rgba(45, 45, 45, 255));
}
QMenuBar::item {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(104, 104, 104, 255), stop:0.464503 rgba(104, 104, 104, 255), stop:0.46856 rgba(90, 90, 90, 255), stop:1 rgba(45, 45, 45, 255));
}
QMenuBar::item:hover, QMenuBar::item:selected {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(124, 124, 124, 255), stop:0.464503 rgba(124, 124, 124, 255), stop:0.46856 rgba(110, 110, 110, 255), stop:1 rgba(65, 65, 65, 255));
border:1px solid #777;
border-radius:2px;
}

QMenu::item {
padding:3px 15px;
border:1px solid #333;
border-radius:2px;
margin:0;
}
QMenu::item:hover,  QMenu::item:selected {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(74,74, 74, 255), stop:0.464503 rgba(74,74,74, 255), stop:0.46856 rgba(60, 60,60, 255), stop:1 rgba(50, 50, 50, 255));
border:1px solid #555;
}

QMenu::separator {
background:#444;
height:2px;
margin:3px 0;
}
QMenu, #menu_File, #menu_Help {
padding:3px;
border:1px solid #333;
border-radius:2px;
margin:-1px;
}

#frame QPushButton {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(104, 104, 104, 255), stop:0.464503 rgba(104, 104, 104, 255), stop:0.46856 rgba(90, 90, 90, 255), stop:1 rgba(45, 45, 45, 255));
         border:1px solid #777;
         border-radius: 3px;
padding:3px 5px;
text-align:center;
color: rgb(200, 200, 200);
}

#frame QPushButton:hover {
                        
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(114, 114, 114, 255), stop:0.464503 rgba(114, 114, 114, 255), stop:0.46856 rgba(100, 100, 100, 255), stop:1 rgba(55, 55, 55, 255));
color: rgb(210,210,210);
}
#frame QPushButton:pressed {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(94,94, 94, 255), stop:0.464503 rgba(94,94, 94, 255), stop:0.46856 rgba(80, 80,80, 255), stop:1 rgba(35, 35, 35, 255));
color: rgb(210,210,210);
         border-color:#666;
}

#centralWidget QScrollBar {
background-color: rgb(50, 50, 50);
color: rgb(200, 200, 200);
border:1px solid #333;
margin:17px 0 17px 0;
}

#centralWidget QScrollBar::handle:vertical {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(104, 104, 104, 255), stop:0.464503 rgba(104, 104, 104, 255), stop:0.46856 rgba(90, 90, 90, 255), stop:1 rgba(45, 45, 45, 255));
color: rgb(200, 200, 200);

                        border:1px solid #444;
border-radius:2px;
}
#centralWidget QScrollBar::handle:vertical:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(114, 114, 114, 255), stop:0.464503 rgba(114, 114, 114, 255), stop:0.46856 rgba(100, 100, 100, 255), stop:1 rgba(55, 55, 55, 255));
border:1px solid #555;
}
#centralWidget QScrollBar::handle:vertical:pressed {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(94,94, 94, 255), stop:0.464503 rgba(94,94, 94, 255), stop:0.46856 rgba(80, 80,80, 255), stop:1 rgba(35, 35, 35, 255));
border:1px solid #555;
}

#centralWidget QScrollBar::add-line:vertical, #centralWidget QScrollBar::sub-line:vertical   {
background-color: rgb(50, 50, 50);
height: 15px;
subcontrol-position: bottom;
subcontrol-origin: margin;
border:1px solid #333;
border-radius:2px;
}
#centralWidget QScrollBar:hover QScrollBar::add-line:vertical, #centralWidget QScrollBar
                        :hover QScrollBar::sub-line:vertical  {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(104, 104, 104, 255), stop:0.464503 rgba(104, 104, 104, 255), stop:0.46856 rgba(90, 90, 90, 255), stop:1 rgba(45, 45, 45, 255));
color: rgb(200, 200, 200);
border:1px solid #444;
border-radius:2px;
}

#centralWidget QScrollBar::sub-line:vertical {
 subcontrol-position: top;
}

#centralWidget  QScrollBar::add-line:vertical:hover,  #centralWidget QScrollBar::sub-line:vertical:hover  {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(114, 114, 114, 255), stop:0.464503 rgba(114, 114, 114, 255), stop:0.46856 rgba(100, 100, 100, 255), stop:1 rgba(55, 55, 55, 255));
border:1px solid #555;
border-radius:2px;
}
#centralWidget QScrollBar::add-line:vertical:pressed,  #centralWidget QScrollBar::sub-line:vertical:pressed  {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(
                        64,64, 64, 255), stop:0.464503 rgba(64,64,64, 255), stop:0.46856 rgba(50,50,50, 255), stop:1 rgba(40, 40, 40, 255));	
}

#centralWidget  QScrollBar::up-arrow:vertical, #centralWidget QScrollBar::down-arrow:vertical {
background:url(:/spacescape/images/bullet_arrow_up.png);
      width: 15px;
      height: 15px;
  }
#centralWidget QScrollBar::down-arrow:vertical {
background:url(:/spacescape/images/bullet_arrow_down.png);
}
#centralWidget  QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
      background: none;
  }
#layerProperties QTreeView {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(22, 22, 22, 255), stop:0.901366 rgba(67, 67, 67, 255), stop:0.960546 rgba(76, 76, 76, 255), stop:1 rgba(88, 88, 88, 255));
alternate-background-color: #1a1a1a;
border:1px solid #000;
color: rgb(200, 200, 200);
}
#helpText {
border:1px solid #444;
color:#999;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QtTreePropertyBrowser" name="layerProperties" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="headerVisible" stdset="0">
            <bool>false</bool>
           </property>
           <property name="resizeMode" stdset="0">
            <string>ResizeToContents</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="newLayer">
              <property name="statusTip">
               <string>Create a new layer</string>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>New Layer</string>
              </property>
              <property name="icon">
               <iconset resource="Spacescape.qrc">
                <normaloff>:/spacescape/images/page_white_add.png</normaloff>:/spacescape/images/page_white_add.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="copyLayer">
              <property name="statusTip">
               <string>Copy the selected layer</string>
              </property>
              <property name="text">
               <string>Copy Layer</string>
              </property>
              <property name="icon">
               <iconset resource="Spacescape.qrc">
                <normaloff>:/spacescape/images/page_white_copy.png</normaloff>:/spacescape/images/page_white_copy.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="moveLayerDown">
              <property name="statusTip">
               <string>Move the selected layer down</string>
              </property>
              <property name="text">
               <string>Move Down</string>
              </property>
              <property name="icon">
               <iconset resource="Spacescape.qrc">
                <normaloff>:/spacescape/images/page_white_put.png</normaloff>:/spacescape/images/page_white_put.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="moveLayerUp">
              <property name="statusTip">
               <string>Move the selected layer up</string>
              </property>
              <property name="text">
               <string>Move Up</string>
              </property>
              <property name="icon">
               <iconset resource="Spacescape.qrc">
                <normaloff>:/spacescape/images/page_white_get.png</normaloff>:/spacescape/images/page_white_get.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>62</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="deleteLayer">
              <property name="statusTip">
               <string>Delete the selected layer</string>
              </property>
              <property name="text">
               <string>Delete Layer</string>
              </property>
              <property name="icon">
               <iconset resource="Spacescape.qrc">
                <normaloff>:/spacescape/images/page_white_delete.png</normaloff>:/spacescape/images/page_white_delete.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="helpText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="indent">
         <number>2</number>
        </property>
       </widget>
      </widget>
      <widget class="TestWidget" name="ogreWindow" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>256</height>
        </size>
       </property>
       <property name="backgroundColor" stdset="0">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="statusTip">
    <string>Open a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>S&amp;ave as...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Spacescape</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>images/page_white_add.png</normaloff>images/page_white_add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="statusTip">
    <string>Clear the scene and start over.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="statusTip">
    <string>Export scene to six images for a skybox</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header>QtTreePropertyBrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TestWidget</class>
   <extends>QWidget</extends>
   <header>TestWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Spacescape.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExport</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onExport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onNewFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newLayer</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onNewLayerClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>copyLayer</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onCopyLayerClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveLayerDown</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onMoveLayerDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveLayerUp</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onMoveLayerUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteLayer</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onDeleteLayerClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onExport()</slot>
  <slot>onNewFile()</slot>
  <slot>onNewLayerClicked()</slot>
  <slot>onCopyLayerClicked()</slot>
  <slot>onMoveLayerUp()</slot>
  <slot>onMoveLayerDown()</slot>
  <slot>onDeleteLayerClicked()</slot>
 </slots>
</ui>
